src/slides/spring-2021-gbm1.yaml

```diff
@@ -0,0 +1,130 @@
+---
+title: Welcome back! SEC GBM 1 Spring 2021
+slides:
+  - content: |
+      ## Things you will need
+
+      - **Create A GitHub account** (github.com)
+      - **Download Git** (git-scm.com)
+      - **Download GitHub CLI** (cli.github.com)
+      - **Download VSCode** (code.visualstudio.com)
+      - **Download Go** (golang.org)
+
+      ## Submit your GitHub username
+
+      We'll send you an invite to the org if you aren't in it, and add you to the team.
+
+      [softe.club/gh-submit](https://softe.club/gh-submit)
+
+      ## Join Slack
+
+      [softe.club/slack](https://softe.club/slack)
+    sticky: true
+  - type: title
+    title: Welcome Back!
+    img: logo
+  - content: |
+      # Open Source Contributions
+      ## Fall 2020
+
+      3 people completed 9 pull requests
+
+      Adam contributed:
+      - https://github.com/StevensSEC/softe.club/pull/191
+      - https://github.com/StevensSEC/softe.club/pull/190
+      - https://github.com/StevensSEC/softe.club/issues/204
+
+      Carson contributed:
+      - https://github.com/radareorg/radare2/pull/17770
+      - https://github.com/mum4k/termdash/issues/249
+
+      Eleni contributed:
+      - https://github.com/dyc3/opentogethertube/issues/311
+      - https://github.com/dyc3/opentogethertube/issues/267
+      - https://github.com/dyc3/opentogethertube/issues/20
+      - https://github.com/dyc3/opentogethertube/issues/273
+  - content: |
+      # Open Source Contributions
+      ## Spring 2021
+      Board for Spring 2021 is now open!
+      If you contribute to an open source project, let us know by adding it to the board!
+
+      https://github.com/orgs/StevensSEC/projects/3
+
+      Link is also on the front page of [soft.club](https://softe.club).
+  - type: title
+    title: What are we making?
+  - type: title
+    title: A Pokemon Battle Library!
+    subtitle: Written in Go
+  - name: clone-instructions
+    content: |
+      ## How to clone the repo
+
+      1. Open a terminal
+        - Powershell on Windows
+      2. Navigate to the folder that you want to place the project into
+        - *It'll automatically create a folder inside the current folder called `pokemonbattlelib`.*
+      3. Run this command:
+
+      ```bash
+      gh repo clone StevensSEC/pokemonbattlelib
+      ```
+      4. Open it in VSCode
+      ```bash
+      cd pokemonbattlelib
+      code .
+      ```
+      *Tip: Press tab to autocomplete.*
+    sticky: true
+  - content: |
+      # What's the goal?
+
+      - Portable library that can be used to add Pokemon battles to anything
+      - No need to worry about accuracy or logic, just implement the frontend
```

Comment:
I feel like this might confuse some club members, maybe specify that the users of the library wont have to worry because our project is the logic